src/python/bcc/__init__.py

```diff
@@ -1093,7 +1093,12 @@ def cleanup(self):
             # non-string keys here include the perf_events reader
             if isinstance(k, str):
```

Comment:
I doubt anyone is using the `return 1` style, the perf_submit functionality was added early enough and all of our tools have adopted it if they could. @brendangregg does that sound accurate to you?